test/e2e/provisioning/testdata/machinedeployment-opennebula.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineDeployment
+metadata:
+  name: << MACHINE_NAME >>
+  namespace: kube-system
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      name: << MACHINE_NAME >>
+  template:
+    metadata:
+      labels:
+        name: << MACHINE_NAME >>
+    spec:
+      providerSpec:
+        value:
+          sshPublicKeys:
+            - "<< YOUR_PUBLIC_KEY >>"
+          cloudProvider: "opennebula"
+          cloudProviderSpec:
+            endpoint: "<< ONE_ENDPOINT >>"
+            username: "<< ONE_USERNAME >>"
+            password: "<< ONE_PASSWORD >>"
+
+            cpu: 1
+            vcpu: 2
+            memory: 1024
+
+            image: "<< OS_IMAGE >>"
```

Comment:
similar to the other comment, can we maybe make sure that this is prefixed to be specific for OpenNebula? e.g. `ONE_OS_IMAGE`? This way we can avoid collision with the OpenStack image reference.